README.md

```diff
@@ -250,17 +250,18 @@ COMP_STMT             --> KEY COMP_OP VALUE
 CONTAINS_STMT         --> KEY not? contains KEY COMP_OP VALUE
 KEY                   --> IDENTIFIER INDEX? (.IDENTIFIER INDEX?)*
 INDEX                 --> [NUMBER]
-IDENTIFIER            --> LETTER+
+IDENTIFIER            --> LETTER (LETTER* DIGIT* UNDERSCORE*)* | UNDERSCORE (LETTER* DIGIT* UNDERSCORE*)*
```

Comment:
`_` would be a valid identifier then. Do we allow this?

---

We could simplify this rule to:

```
IDENTIFIER --> (LETTER|UNDERSCORE) (LETTER* DIGIT* UNDERSCORE*)*
````

Reply:
Technically, I see no problem with allowing this, so I would allow it.
Thanks for the simplification of the grammar statement